lib/contracts/create3/utils.go

```diff
@@ -0,0 +1,13 @@
+package create3
+
+import (
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+// HashSalt returns the [32]byte hash of the salt string.
+func HashSalt(s string) [32]byte {
```

Comment:
maybe a test for this? might be overkill

Reply:
Do you know any good keccak hashes I could use

Reply:
I don't think we need to test the Go `crypto` package. However, we might have a simple test to verify if the hashing algorithm(s) have changed. E.g. use one well known input and compare against its well known output as a sanity check.

Reply:
found one

Reply:
yeah did that